dynamo/main.tf

```diff
@@ -0,0 +1,18 @@
+resource "aws_dynamodb_table" "table" {
+  name           = var.table_name
+  read_capacity  = var.read_capacity
+  write_capacity = var.write_capacity
```

Comment:
Is there a reason we're provisioning our ddb table capacity rather than using `PAY_PER_REQUEST`? I must admit I've not done a pricing comparison myself but are we expecting our tables to be greatly used?

Reply:
Mostly because I don't know how Dynamo works. I've changed it to take a variable for billing_mode and defaulted it to pay per request and then only set the read and write capacity if it isn't set to pay_per_request. That way, we can create a provisioned table with this module but it will default to pay_per_request.